### PR TITLE
CFE-4177: Changed style for commands and excluded $

### PR DESCRIPTION
**Pull request by nickanderson**

### CODE REVIEW

**Comment by nickanderson** [comment]
@cf-bottom jenkins, please

**Comment by cf-bottom** [comment]
Alright, I triggered a build:

[Build Status](https://ci.cfengine.com//job/fast-build-and-deploy-docs-master/339/)

**Jenkins:** https://ci.cfengine.com/job/fast-build-and-deploy-docs-master/339/

**Documentation:** http://buildcache.cfengine.com/packages/build-documentation-pr/jenkins-fast-build-and-deploy-docs-master-339/output/_site/

File: getting-started/installation/general-installation/installation-community.markdown [hunk 1]
```diff
@@ -26,14 +26,15 @@ Please Note: Internet access is required from the host if you wish to use the qu
 
 Use the following script to install CFEngine on your 32- or 64-bit machine.
 
-```
-$ wget -O- http://cfengine.package-repos.s3.amazonaws.com/quickinstall/quick-install-cfengine-community.sh | sudo bash
+```console
+wget -O- http://cfengine.package-repos.s3.amazonaws.com/quickinstall/quick-install-cfengine-community.sh | sudo bash
```
Comment (olehermanse) [inline getting-started/installation/general-installation/installation-community.markdown:30]: Shouldn't these be `command`?

Comment (olehermanse) [reply getting-started/installation/general-installation/installation-community.markdown:30]: I am testing this out here: https://github.com/cfengine/documentation/pull/3028

**Pull request merged**